jormungandr-integration-tests/mock/proto/node.proto

```diff
@@ -0,0 +1,144 @@
+syntax = "proto3";
```

Comment:
Would it be possible to use the source of truth .proto file from its chain-deps location, by relative path? The copy creates a maintenance burden.